crates/solidity/testing/sanctuary/src/datasets.rs

```diff
@@ -61,6 +61,12 @@ impl DataSet {
     }
 
     pub fn checkout_directory(&self, directory: &str) -> Result<&Vec<SourceFile>> {
+        Command::new("git")
```

Comment:
Do we make the directory dirty as part of the sanctuary run? If so, should we also run a `git clean` as well?

Reply:
Nop. But for local development, I often ended up needing to edit these files in place, to experiment with our error reporting/locations.
I suspect others will be doing the same as well, so I added it for extra insurance. I will add a comment 👍